configuration/src/main/java/io/camunda/configuration/Disk.java

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.configuration;
+
+public class Disk {
```

Comment:
@igpetrov  There are two missing properties here. monitoring-enabled and monitoring-interval. See `io.camunda.zeebe.broker.system.configuration.DiskCfg`. Not sure how we missed them to add in the spread sheet. Could you please also add those properties?

Reply:
@deepthidevaki added to PR, ticket, and the doc. Thanks for the good catch. I should have spotted that as well. I'll be more careful next time.